processing/src/main/java/io/druid/query/Druids.java

```diff
@@ -462,13 +463,9 @@ public TimeseriesQueryBuilder filters(String dimensionName, String value)
       return this;
     }
 
-    public TimeseriesQueryBuilder filters(String dimensionName, String value, String... values)
```

Comment:
I believe the old impl would actually have thrown an exception in this case. It would have tried to construct an empty OrFilter, which isn't allowed.